rfcs/053-filtering-by-subject/README.md

```diff
@@ -0,0 +1,101 @@
+# RFC 053: Filtering by contributor, genre, and subject
+
+For per-subject and per-person pages, we want to filter for images and works that match a given subject/person.
+For example:
+
+*   Works by Florence Nightingale
+*   Works about Charles Darwin
+*   Images about mental health
+
+This turns out to be non-trivial, so this RFC describes how we'll get there.
+
+Strictly speaking we only need subjects and contributors for this work, but genres are so similar we should treat them in the same way.
+
+## Requirements
+
+1.  On per-concept pages, there's a sample of matching images/works.
+
+2.  On per-concept pages, there's a link to a filtered search for the given concept, directly below the sample results.
+
+3.  On work pages, the list of subjects/contributors/genres to link to filtered searches for each subject.
```

Comment:
On works pages, the list of subjects/contributors/genres to link to concept pages.

Reply:
Clarified.